packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/FloatExtensibleEnum.cs

```diff
@@ -0,0 +1,58 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace UnbrandedTypeSpec.Models
+{
+    public readonly partial struct FloatExtensibleEnum : IEquatable<FloatExtensibleEnum>
+    {
+        private readonly float _value;
+        private const float OneValue = 1F;
```

Comment:
can we add 1.1 as an option as well?  Strange that FloatFixedEnum uses 1.1 and this one uses 1

Reply:
well we might be intentionally doing this to verify that we could distinguish the enums with int and float.

I will add another one

Reply:
Added